### DISCUSSION TITLE
Can't generate JWT config

### DISCUSSION THREAD

**Discussion by arkapratimc** [hasura/graphql-engine#9292] [General]
After selecting Auth0 [here](https://hasura.io/jwt-config/), it's returning nothing, bug?

https://user-images.githubusercontent.com/105232141/207095583-b76c256b-5deb-403d-a5e7-369b717ecdef.mp4

**Comment by adas98012** [comment]
@arka1002 
It might have been a temporary UI issue, but you should see a field to enter the "Auth0 domain name": 

<img width="648" alt="Screen Shot 2022-12-14 at 12 40 06 PM" src="https://user-images.githubusercontent.com/101733366/207709259-c25b5ebd-6a5f-408f-94d0-8a62f4775dc9.png">

**Reply by arkapratimc** [reply]
ya, it's working alright
